server/models/classes.py

```diff
@@ -0,0 +1,137 @@
+    'http://localhost:5004'
+]
+
+
+class Lokals():
+
+    def __init__(self):
+        self.apartments = []
+        self.load_apartment_from_database()
+
+    def create_apartments(self, name):
+        self.apartments.append(Apartment(name))
+
+    def load_apartment_from_database(self):
+        with CursorFromConnectionPool() as cursor:
+            cursor.execute('SELECT apartment_name FROM apartments')
+            apartments_data = cursor.fetchall()
+            for apartment in apartments_data:
+                self.create_apartments(apartment[0])
+
+
+class Apartment:
+
+    def __init__(self, name):
+        self.name = name
+        self.rooms = []
+
+    def create_room(self, name):
+        self.rooms.append(Room(name))
+
+    def load_rooms(self):
+        with CursorFromConnectionPool() as cursor:
+            cursor.execute(
+                'SELECT name FROM rooms WHERE id_apartment = (SELECT id FROM apartments WHERE apartment_name = (%s));', (self.name,))
+            room_data = cursor.fetchall()
+            for room in room_data:
+                self.create_room(room[0])
+
+
+class Room:
+    def __init__(self, name):
+        self.name = name
+        self.devices = []
+
+    def create_device(self, address, id, name, type_device):
+        self.devices.append(Device(address, id, name, type_device))
+
+    def load_devices(self):
+        with CursorFromConnectionPool() as cursor:
+            cursor.execute(
+                'SELECT address, id, name, type FROM devices WHERE id_rooms = (SELECT id FROM rooms WHERE name = (%s));', (self.name,))
+            device_data = cursor.fetchall()
+            for device in device_data:
+                self.create_device(device[0], device[1], device[2], device[3])
+
+
+class Device:
+
+    def __init__(self, address=None, id=None, name=None, type_device=None):
+        self.id = id
+        self.name = name
+        self.address = address
+        self.type_device = type_device
+
+    def creata_device(self):
+        return self()
+
+    def find_in_database(self):
+        resp = requests.get(f'{self.address}/nameandtype').json()
+        self.name, self.type_device = resp
+        with CursorFromConnectionPool() as cursor:
+            cursor.execute(
+                'SELECT id FROM devices WHERE name = %s AND type = %s AND address = %s;', (self.name, self.type_device, self.address))
+            id_data = cursor.fetchall()
+        if id_data:
+            self.id = id_data[0]
+            requests.get(f'{self.address}/{self.id}')
+        else:
+            return True
+
+    def add_device_to_database(self, apartment, room):
+        with CursorFromConnectionPool() as cursor:
+            cursor.execute('INSERT INTO devices( name , address, type ) VALUES  (%s, %s, %s)  RETURNING id',
+                           (self.name, self.address, self.type_device))
+            self.id = cursor.fetchone()[0]
+        with CursorFromConnectionPool() as cursor:

```

Comment:
You use this fragment of code few times now. Make it generic static method, and wrap these methods into class.